### DISCUSSION TITLE
[Question][Hw3] Unbound value Instrument.run

### DISCUSSION THREAD

**Discussion by 0xXcrypted** [prosyslab-classroom/cs348-information-security#546] [Q&A]
I faced a new kind of error when running make command.
`Error: Unbound value Instrument.run`

This error is raised in src/main.ml, Can anyone help me with this?
Thanks

**Comment by mickowale** [comment]
You may have changed the name of `run` function in instrument.ml.

**Comment by 0xXcrypted** [comment]
I’ve checked that but that is not my case.
Thanks for your suggestion

**Comment by RiceBiscuits** [comment]
Please check that `run` function is complete.

**Comment by miraliahmadli** [comment]
If you are doing some recursive calls, that may have caused the issue: https://stackoverflow.com/questions/26491703/ocaml-error-unbound-value-when-using-recursion

If you are not doing recursively, just ignore this comment

**Comment by 0xXcrypted** [comment]
Reconnecting to the VM solved this problem.
Thanks everyone
